spinalcordtoolbox/download.py

```diff
@@ -46,8 +46,9 @@
     },
     "PAM50": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20230519/PAM50-r20230519.zip",
-            "https://osf.io/8cgme/?action=download",
+            # DO NOT MERGE. This is a prerelease. This should be superseded by a new PAM50 link once spinal level changes are merged.
+            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20231011/PAM50-r20231011.zip",
```

Comment:
Note to self: The PAM50 template will be properly updated in a separate PR. We need to make sure _that_ URL is also added to this PR, rather than the temporary prerelease I created above.

Reply:
As soon as https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4268 is merged, I will revert the PAM50 related changes from this PR.